lib/index.js

```diff
@@ -37,10 +38,9 @@ Amplitude.prototype.group = group;
 function track(payload, fn){
   var self = this;
   return this
-    .get('/httpapi')
-    .type('json')
-    .query({ api_key: this.settings.apiKey })
-    .query({ event: JSON.stringify(payload) })
+    .post('/httpapi')
+    .send('api_key=' + this.settings.apiKey)
```

Comment:
I tried that but it returns a bad request. @djih , are you sure that you all accept JSON requests? the format Han suggests will send over a JSON POST request. I'm thinking this is failing because Amplitude expects an encoded string.